commercetools-models/src/main/java/io/sphere/sdk/apiclient/commands/updateactions/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ {@doc.gen summary api client}
+ */
+package io.sphere.sdk.apiclient.commands.updateactions;
```

Comment:
Do we really need this if we don't have any `ApiClient` update actions?

Reply:
Yep this can be removed, but the folder have to remain, otherwise we ll have a javadoc issue.